### ISSUE TITLE
PHP Warning: unpack() on ini mbstring.func_overload = 2 setting

### ISSUE THREAD

**Issue by Cramac** [mpdf/mpdf#91]
if i enable in php.ini
mbstring.func_overload = 2
when you try to generate a pdf I receive an error
PHP Warning:  unpack() [<a href='function.unpack'>function.unpack</a>]: Type n: not enough input, need 2, have 0 in /var/www/zoostaff/data/www/base/inc/mpdf/classes/ttfontsuni.php on line 3817, referer:

**Comment by finwe** [comment]
Could you please provide more details? What is the content of your file? Are you using custom fonts? If so, which ones?

**Comment by Cramac** [comment]
i use this:
$mpdf = new mPDF('utf-8', 'A4', '8', '', 10, 10, 7, 7, 10, 10);
$mpdf->charset_in = 'utf-8'; 
$mpdf->list_indent_first_level = 0; 
$mpdf->WriteHTML($out, 2); 
$mpdf->Output('mpdf.pdf', 'I');

the content of the page is a table and Russian text
and i set mbstring.func_overload = 2
i have error

**Comment by bunya** [comment]
Hello!
We had same problem time ago. It's due to some single-byte string functions using as multi-byte ones. It's broke internal logic (if not mistaken - checking font element position).
We solved the problem step by step:
In first step we transfered mpdf generation to separate php-fpm process (there mbstring.func_overload = 0). In second - we cleaned our code, separated by logic single-byte string functions and multi-byte ones and set mbstring.func_overload = 2
